suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -962,3 +963,18 @@ export const readUtxoOutpoint = (outpoint: string) => {
 
 export const isSameUtxo = (a: AccountUtxo, b: AccountUtxo) =>
     a.txid === b.txid && a.vout === b.vout;
+
+/**
+ * Returns true if network uses receive address instead of XPUB.
+ */
+export const isAddressBasedNetwork = (networkType: NetworkType) => {
```

Comment:
Woudn't be better to have this in as bool in networkConfig?

Reply:
Why not, do you think so?